src/riscv/syscall/ret.rs

```diff
@@ -0,0 +1,38 @@
+// Since ckb-vm can only return 0 or 1 as exit code, We must find another way to return string, u64...
+use std::cell::RefCell;
+use std::rc::Rc;
+
+use ckb_vm::instructions::Register;
+
+use crate::riscv::syscall::common::get_arr;
+use crate::riscv::syscall::convention::SYSCODE_RET;
+
+pub struct SyscallRet {
+    data: Rc<RefCell<Vec<u8>>>,
```

Comment:
`Rc<RefCell>` seems unnecessary because `ecall` is `mut`.

Reply:
The SyscallRet.data is initialized by riscv interpreter, and the interpreter needs to read it. See: https://github.com/mohanson/cita-vm/blob/riscv2/src/riscv/interpreter.rs#L52

Reply:
I see